tiles/src/main/java/com/protomaps/basemap/layers/Water.java

```diff
@@ -51,25 +102,83 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
         .setMinPixelSize(3.0)
         .setBufferPixels(8);
 
+      String kind = "other";
+      String kind_detail = "";
+      var reservoir = false;
+      var alkaline = false;
+
+      // coallese values across tags to single kind value
+      if (sf.hasTag("natural", "water", "bay", "strait", "fjord")) {
```

Comment:
Use the same upgraded `kind` logic across water, physical lines, and physical points (label centroids)